apps/heft/src/utilities/TaskPackageResolver.ts

```diff
@@ -222,7 +248,7 @@ export class TaskPackageResolver {
     if (!visitedRigPackagePaths.has(rigPackagePath)) {
       visitedRigPackagePaths.add(rigPackagePath);
 
-      const rigPackageJson: INodePackageJson = TaskPackageResolver._packageJsonLookup.loadNodePackageJson(
+      const rigPackageJson: INodePackageJson = this._packageJsonLookup.loadNodePackageJson(
```

Comment:
Obligatory 'more async'

Reply:
Same here - there isn't an async variant of `loadNodePackageJson`.